packages/plugin-core/src/commands/base.ts

```diff
@@ -50,6 +51,11 @@ export abstract class BaseCommand<
 
   static showInput = window.showInputBox;
 
+  /**
+   * Does this command require an active workspace in order to function
+   */
+  static requireActiveWorkspace = false;
```

Comment:
I think this is fine for this PR, but I think we should eventually switch this to true. A lot of our code relies on an active workspace existing, so it would be safer to only set this to false for commands we know are safe without a workspace.